com.airtame.Client.json

```
@@ -1,10 +1,10 @@
 {
   "app-id": "com.airtame.Client",
   "runtime": "org.freedesktop.Platform",
-  "runtime-version": "19.08",
+  "runtime-version": "20.08",
```

I guess this can be bumped to 21.08?
```suggestion
  "runtime-version": "21.08",
```

---

Yep thanks missed that.